.github/workflows/bandit.yml

```diff
@@ -0,0 +1,37 @@
+name: Bandit
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'src/bindings/python/**'
+      - 'samples/python/**'
+      - 'tools/**'
+      - 'scripts/utils/**'
+      - 'docs/**'
+      - '.ci/azure/**'
+  pull_request:
+    paths:
+      - 'src/bindings/python/**'
+      - 'samples/python/**'
+      - 'tools/**'
+      - 'scripts/utils/**'
+      - 'docs/**'
+      - '.ci/azure/**'
+jobs:
+  bandit_check:
+    runs-on: ubuntu-18.04
```

Comment:
if possible we should use ubuntu 20 because ubutnu 18 LTS support will end soon.

Reply:
Fixed in last commit